modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/A11yTool.testcase

```diff
@@ -413,6 +415,36 @@ definition {
 				locator1 = "A11yTool#HIGHLIGHTED_ELEMENT_POPOVER_IMPACT");
 		}
 
+		task ("Unfilter impact by Critical") {
+			A11yTool.setFilter(filterType = "Critical");
+
+			FormFields.viewCheckboxNotChecked(fieldName = "Critical");
+		}
+
+		task ("Assert page has Minor, Moderate, Serious, and Critical set of violations") {
+			AssertElementPresent(
+				key_impact = "minor",
+				locator1 = "A11yTool#VIOLATION_PANEL_ITEM_IMPACT");
+
+			AssertElementPresent(
+				key_impact = "moderate",
+				locator1 = "A11yTool#VIOLATION_PANEL_ITEM_IMPACT");
+
+			AssertElementPresent(
+				key_impact = "serious",
+				locator1 = "A11yTool#VIOLATION_PANEL_ITEM_IMPACT");
+
+			AssertElementPresent(
+				key_impact = "critical",
+				locator1 = "A11yTool#VIOLATION_PANEL_ITEM_IMPACT");
+		}
+
+		task ("Assert violations are highlighted with an icon") {
+			AssertElementPresent(locator1 = "A11yTool#HIGHLIGHTED_ELEMENT");
```

Comment:
For my understanding, this locator matches multiple elements in the page, and the assertion always find out the first one, but just looking at the test, it cannot tell the first one is which level of violation. So I think it should be better to assert some specific elements of all levels of violations. Similar as these lines: https://github.com/liferay-frontend/liferay-portal/blob/867c433fa4948d3882ba3952fc363e416e2b1a9a/modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/A11yTool.testcase#L397-L399 but assert all levels of violations.